src/common/iron-session/web-iron-session-provider.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  IronSessionProvider,
+  SealDataOptions,
+  UnsealedDataType,
+} from './iron-session-provider';
+
+/**
+ * WebIronSessionProvider which uses the base iron-session seal/unseal methods.
+ */
+export class WebIronSessionProvider extends IronSessionProvider {
+  /** @override */
+  async sealData(data: unknown, options: SealDataOptions): Promise<string> {
+    const ironSession = await import('iron-session');
+    return ironSession.sealData(data, options);
```

Comment:
Using dynamic imports here because the problematic crypto import is in `iron-session`'s index.js.